mobify/platform-scaffold
# [Integration Manager] Build in more of the Login component

Subsequent to discussions about hard-coding versus parsing content, this PR modifies the Login container to explicitly specify the form fields, similar to the various checkout forms. The increased simplicity this brought to the components allowed substantial code removal.

 **JIRA**: N/A

## Changes
- Build in the shape of the signin and register forms
- No longer parse that content from the HTML in the Merlin's connector
- Convert the ui -> login branch of the store to a simple collection of flags
- Move the signin vs register page determination from the connector to the component
- General minor refactors of the code

## How to test-drive this PR
- Run the tests
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that the login page loads and runs as before (but faster-looking!)
- Do the same with the Demandware site.
